src/current/v23.1/molt-verify.md

```diff
@@ -9,15 +9,15 @@ docs_area: migrate
 {% include feature-phases/preview.md %}
 {{site.data.alerts.end}}
 
-The MOLT Verify tool checks for data discrepancies between a source and a target database during a [database migration](migration-overview.html).
```

Comment:
i think you can defer this change until we ship a new release (not done yet).

Reply:
Reverting most of the changes. When is the release expected?